map_gui/src/misc_tools.rs

```diff
@@ -0,0 +1,201 @@
+use geom::{ArrowCap, Distance};
+use map_model::{LaneID, TurnType};
+use widgetry::{
```

Comment:
Just a naming nit, crate modules like "helpers" and "misc_tools" aren't great for discoverability.

This could just be "turn_explorer.rs", not a big deal though.

Reply:
Agreed! I'm preserving the bad names from `game` to make moving things around a little easier, but I'm almost at the cleanup point. Going to flatten things out / rename more reasonably.